dev/integration_tests/ui/pubspec.yaml

```diff
@@ -2,6 +2,7 @@ name: integration_ui
 description: Flutter non-plugin UI integration tests.
 
 dependencies:
+  image: 1.1.29
```

Comment:
please make sure to run `flutter update-packages --force-upgrade` when adding dependencies

Reply:
Interesting. We already depend on `image: 1.1.29` in another package. This change has no intention to upgrade our packages to new versions. Perhaps we need another mode in the `flutter update-packages` command that completes the list of transitive dependencies without upgrading?